DispatchR/DispatchR.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;net5.0;net6.0;net7.0;</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;net5.0;</TargetFrameworks>
```

Comment:
If you change this does the future frameworks show up as supported in Nuget package manager?